src/components/MCQ.vue

```diff
@@ -1,9 +1,18 @@
 <template>
   <div>MCQ Test</div>
   <div class="mcq-title">{{ title }}</div>
+  <ul>
+    <li
+      v-for="[key, value] in Object.entries(options)"
+      :key="key"
+      class="mcq-option"
+    >
+      {{ key }}: {{ value.text }}
+    </li>
+  </ul>
```

Comment:
I wouldn't use the `key` construct. You are starting to tie yourself to an implemenation detail.